### PR TITLE
Ported SQL to NET from Java.

### PR DESCRIPTION
**Pull request by anudeepsharma**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@anudeepsharma__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by cormacpayne** [comment]
@hovsepm Hey Hovsep, can you assign this to someone working on fluent?

**Comment by hovsepm** [comment]
> using Microsoft.Azure.Management.Resource.Fluent.Authentication;

Please add license header. 

---
Refers to: src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/SqlManager.cs:1 in 2797994. [](commit_id = 2797994c8887161dd4d76e42ecfe4e3215fbfbec, deletion_comment = False)

File: src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/Domain/InterfaceImpl/ElasticPoolActivityImpl.cs [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+
+    internal partial class ElasticPoolActivityImpl 
+    {
+        /// <return>The name of the resource.</return>
+        string Microsoft.Azure.Management.Resource.Fluent.Core.IHasName.Name
+        {
+            get
+            {
+                return this.Name() as string;
```
Comment (hovsepm) [inline src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/Domain/InterfaceImpl/ElasticPoolActivityImpl.cs]: >as string [](start = 35, length = 9)

no need to convert string to string.

File: src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/Domain/InterfaceImpl/ElasticPoolActivityImpl.cs [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+            }
+        }
+
+        /// <return>The resource ID string.</return>
+        string Microsoft.Azure.Management.Resource.Fluent.Core.IHasId.Id
+        {
+            get
+            {
+                return this.Id() as string;
+            }
```
Comment (hovsepm) [inline src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/Domain/InterfaceImpl/ElasticPoolActivityImpl.cs]: no need to convert string to string.

File: src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/Domain/InterfaceImpl/ElasticPoolActivityImpl.cs [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+        }
+
+        /// <return>The name of the resource group.</return>
+        string Microsoft.Azure.Management.Resource.Fluent.Core.IHasResourceGroup.ResourceGroupName
+        {
+            get
+            {
+                return this.ResourceGroupName() as string;
+            }
```
Comment (hovsepm) [inline src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/Domain/InterfaceImpl/ElasticPoolActivityImpl.cs]: no need to convert string to string.

File: src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/Domain/InterfaceImpl/ElasticPoolActivityImpl.cs [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+        }
+
+        /// <return>The current state of the operation.</return>
+        string Microsoft.Azure.Management.Sql.Fluent.IElasticPoolActivity.State
+        {
+            get
+            {
+                return this.State() as string;
+            }
```
Comment (hovsepm) [inline src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/Domain/InterfaceImpl/ElasticPoolActivityImpl.cs]: no need to convert string to string.

File: src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/Domain/InterfaceImpl/ElasticPoolActivityImpl.cs [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+        }
+
+        /// <return>The name of the Azure SQL Server the Elastic Pool is in.</return>
+        string Microsoft.Azure.Management.Sql.Fluent.IElasticPoolActivity.ServerName
+        {
+            get
+            {
+                return this.ServerName() as string;
+            }
```
Comment (hovsepm) [inline src/ResourceManagement/Sql/Microsoft.Azure.Management.Sql.Fluent/Domain/InterfaceImpl/ElasticPoolActivityImpl.cs]: no need to convert string to string.

**Review by hovsepm — CHANGES_REQUESTED** [review]
Please fix license header comment. The rest should be addressed on the converter side. I've created an issue there https://github.com/Microsoft/java-to-csharp-api-generator/issues/29

**Review by hovsepm — APPROVED** [review]

**Pull request merged**